package.json

```diff
@@ -30,7 +30,8 @@
     "build:rust": "cargo build --workspace && npm run postbuild-cargo",
     "postbuild-cargo": "node scripts/copy-rename.js",
     "lint": "eslint \"./src/**/*.ts\"",
-    "format-check": "prettier --check \"./src/**/*.ts\"",
+    "format-check": "prettier --check './src/**/*.ts'",
+    "format-fix": "prettier --write './src/**/*.ts'",
```

Comment:
@TieDyedSheep  I think these might need the escaped quotes like in the `lint` for this to work on windows. I am currently on my mac so I can't verify this

Reply:
I just updated it